content/news/new-project-logo.md

```diff
@@ -0,0 +1,43 @@
+Today we're pleased to show off a project that we have been working on for some time: a new logo for OpenRA!
+
+<div style="text-align:center" markdown="1">
+![](/images/news/20161223-new-logo.png)
+</div>
+
+This logo will be used to represent the OpenRA project as a whole, with the Soviet star logo being kept for the Red Alert mod.
+If community feedback is positive then we plan to roll this out with the next OpenRA release.
+
+Love it? Hate it? Let us know what you think in the comments below!
+
+### How does it look?
```

Comment:
I'd probably move this to the end of that short faq. After the images, no-one's going to read the rest of the post.